youtube_dl/extractor/pbs.py

```diff
@@ -666,25 +666,18 @@ def extract_redirect_urls(info):
         age_limit = US_RATINGS.get(rating_str)
 
         subtitles = {}
-        closed_captions_url = info.get('closed_captions_url')
```

Comment:
Do not remove the old code.

Reply:
So you want me to keep the old subtitle download logic alongside the new one?

Reply:
If I understood you correctly, then my latest commit should be ok.